### DISCUSSION TITLE
3.3.5 breaking x-show enter transitions

### DISCUSSION THREAD

**Discussion by aaronjpitts** [alpinejs/alpine#2094] [7. Bugs (archived)]
Hey Caleb,

3.3.5 is breaking x-show enter transitions, here is a working example of an offcanvas menu working in 3.3.4: https://codepen.io/aaronjpitts/pen/xxrWNXO

In 3.3.5 you will see it's now not using the enter transitions: https://codepen.io/aaronjpitts/pen/WNOzBjY

Cheers,
Aaron

**Comment by SimoTod** [comment]
Hi @aaronjpitts 

Can you try this version and let me know if it works correctly https://codepen.io/SimoTod/pen/bGRMENw ?

FYI, this project does not use the issue tab: https://twitter.com/calebporzio/status/1430564014464159749 (GitHub doesn't allow to remove "create issue from discussion" but it's expected that new bugs are initially created as a discussion).

**Reply by aaronjpitts** [reply]
Hey @SimoTod ,

I can confirm it works :)

Yes, I've realised now to use the discussion tab. Will do going forward if needed.

Cheers,
Aaron
